modules/apps/transfer/types/transfer_authorization.go

```diff
@@ -9,10 +14,11 @@ import (
 	host "github.com/cosmos/ibc-go/v6/modules/core/24-host"
 )
 
-const gasCostPerIteration = uint64(10)
```

Comment:
Removed this in favour of `ctx.KVGasConfig().IterNextCostFlat` below. Considering v6.2.0 is the introduction of the authz feature to the v6 line I thought it was best to align with what is on main.